src/submission/error-handler.service.ts

```diff
@@ -172,9 +172,11 @@ export class ErrorHandlerService {
       );
 
       // Email subject
-      const processCode = emailTemplateContextForUser.processCode || 'N/A';
-      const emailSubject = this.buildEmailSubject(processCode, emailTemplateContextForUser.orisCode, emailTemplateContextForUser.configuration);
-
+    const processCode = emailTemplateContextForUser?.processCode || 'N/A';
+    const env = this.configService.get<string>('app.env')?.trim()?.toLowerCase();
+    const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? ` (sent from ECMPS 2.0 ${env})` : '(sent from ECMPS 2.0)';
```

Comment:
The final template already has a space, the space before the parenthesis here is extra:
```suggestion
    const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? `(sent from ECMPS 2.0 ${env})` : '(sent from ECMPS 2.0)';
```